moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -50,6 +51,73 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+// todo: move elsewhere?
+const COMPONENT_DEFINITIONS = {
```

Comment:
This is not future proof. Alternatives:
- Query component types from DB and use as a lookup
- ?

Reply:
➕ Agreed. This will almost certainly have to change to a DB lookup, we are creating/editing types in the future that will affect their name and index id.

Reply:
👍 thanks for the nudge. i'll re-use a query from ProjectComponents/